### PR TITLE
Fix FSharp tests

### PR DESCRIPTION
**Pull request by Frassle**
This fixes the FSharp tests so that, they run and pass and are also included by build.ps1 to run in CI.

### CODE REVIEW

**Comment by superyyrrzz** [comment]
@Frassle test failed on CI:

> The test source file "D:\a\1\s\test\Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests\bin\Release\net462\TestData\NetCoreLibProject\bin\Release\netstandard2.0\NetCoreLibProject.dll" provided was not found.

**Comment by Frassle** [comment]
Ah I think I know the issue, I'll fix it over the weekend. Just need to exclude the test data projects from the test list.

File: test/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests.fsproj [hunk 1]
```diff
@@ -6,11 +6,21 @@
     <Compile Include="FSharpProjectTests.fs" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="TestData/*/*.fsproj;TestData/**/*.fs">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+  <Target Name="RestoreTestProjects" BeforeTargets="Build">
```
Comment (Frassle) [inline test/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests.fsproj]: I'm not an MSBuild expert so there may be a better way of doing a restore then copy but I could only get this working reliably via two targets.

**Comment by superyyrrzz** [comment]
Thank you @Frassle 
/cc @928PJY

**Pull request merged**